dotCMS/src/main/java/com/dotmarketing/portlets/workflows/actionlet/ResetApproversActionlet.java

```diff
@@ -0,0 +1,125 @@
+
+package com.dotmarketing.portlets.workflows.actionlet;
+
+import com.dotcms.util.CollectionsUtils;
+import com.dotmarketing.business.APILocator;
+import com.dotmarketing.exception.DotDataException;
+import com.dotmarketing.exception.DotSecurityException;
+import com.dotmarketing.portlets.contentlet.model.Contentlet;
+import com.dotmarketing.portlets.workflows.model.*;
+import com.dotmarketing.util.Logger;
+import com.dotmarketing.util.UUIDUtil;
+import com.dotmarketing.util.UtilMethods;
+import com.hazelcast.util.CollectionUtil;
+
+import java.util.*;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+import static com.dotmarketing.portlets.workflows.util.WorkflowActionletUtil.getParameterValue;
+
+/**
+ * Deletes the approvers of the workflow task
+ * @author jsanca
+ */
+public class ResetApproversActionlet extends WorkFlowActionlet {
+
+	private static final String ID_DELIMITER = ",";
+	private static final String PARAM_CONTENT_ACTIONS = "action";
+
+	private static ArrayList<WorkflowActionletParameter> ACTIONLET_PARAMETERS = null;
+
+	@Override
+	public synchronized List<WorkflowActionletParameter> getParameters() {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use block level rather than method level synchronization](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=4904110)